### ISSUE TITLE
Onnx export in different version behaves differently, BatchNorm operator has multi output 

### ISSUE THREAD

**Issue by SystemErrorWang** [pytorch/pytorch#71394]
### 🐛 Describe the bug

I use the following code to export onnx model, which contains Adain op (https://github.com/NVlabs/MUNIT/blob/master/networks.py)
In pytorch 1.8.0, the batchnorm op behaves normal, but the onnxruntime results are different from the pytorch results; In pytorch 1.10.1, the onnxruntime can generate the same results as pytorch, but the batchnorm op has 5 output when visualized by Netron:
<img width="1072" alt="test" src="https://user-images.githubusercontent.com/26077659/149864522-db689d4f-bf56-4331-8be5-4141c5641818.png">

```
image1 = torch.randn(1, 3, 224, 192)
image2 = torch.randn(1, 3, 112, 112)
model = FullModel(num_ch=28)

weight = torch.load('model.pth', map_location='cpu')
model.generator.load_state_dict(weight['g'])
#model.generator.eval()
model.encoder.load_state_dict(weight['e'])
#model.encoder.eval()
model.eval()
with torch.no_grad():
    torch.onnx.export(model, (image1, image2), 'test.onnx', opset_version=11)
```

### Versions

Python platform: Darwin-19.6.0-x86_64-i386-64bit
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.19.2
[pip3] numpydoc==1.1.0
[pip3] pytorch-fid==0.2.0
[pip3] torch==1.10.1
[pip3] torchfile==0.1.0
[pip3] torchvision==0.11.2
[conda] _tflow_select             2.3.0                       mkl  
[conda] blas                      1.0                         mkl  
[conda] mkl                       2019.4                      233  
[conda] mkl-service               2.3.0            py36hfbe908c_0  
[conda] mkl_fft                   1.0.14           py36h27c97d8_0  
[conda] mkl_random                1.0.4            py36h27c97d8_0  
[conda] numpy                     1.19.2                   pypi_0    pypi
[conda] numpy-base                1.17.0           py36h6575580_0  
[conda] numpydoc                  1.1.0                      py_0  
[conda] pytorch-fid               0.2.0                    pypi_0    pypi
[conda] tensorflow                1.13.1          mkl_py36haf07a9b_0  
[conda] tensorflow-base           1.13.1          mkl_py36hc36dc97_0  
[conda] torch                     1.10.1                   pypi_0    pypi
[conda] torchfile                 0.1.0                    pypi_0    pypi
[conda] torchvision               0.11.2                   pypi_0    pypi

**Comment by SystemErrorWang** [comment]
some update:
I used cleanup function from ONNX GraphSurgeon in tensor RT toolkit to remove the unused graph inputs and output. before cleanup, the onnx runtime can generate outputs exactly the same as pytorch. However, after cleanup, the onnx runtime generates different output.

**Comment by garymm** [comment]
`1.10.1, the onnxruntime can generate the same results as pytorch, but the batchnorm op has 5 output when visualized by Netron`

I don't understand the problem. If ONNX Runtime can use the model successfully, what exactly is the problem?
Please be very explicit about the difference between correct behavior and observed behavior.
